tasks/refresh.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../../ruby_task_helper/files/task_helper.rb'
+require_relative '../lib/cli_helper.rb'
+
+class TerraformOutput < TaskHelper
```

Comment:
The references to `output` throughout this file should be `refresh`

Reply:
Thanks for the catch.